blockchain/fullblocktests/generate.go

```diff
@@ -1695,6 +1811,111 @@ func Generate(includeLargeReorg bool) (tests [][]TestInstance, err error) {
 	})
 	rejected(blockchain.ErrStakeTxInRegularTree)
 
+	// Create block with an invalid coinbase transaction.
```

Comment:
```Go
// Create block with a fraudulent coinbase transaction (invalid height).
```

Also, this should be under the malformed coinbase tests next to the related invalid index test (bcb4, which would make this one bcb4a).